bin/metadata.js

```diff
@@ -3,36 +3,57 @@
 
 const fs = require('fs');
 const path = require('path');
+const argv = require('yargs').
```

Comment:
Ah..just wondered why travis passed with this style, turns out we do not run `npm run lint` in travis.yml...can you run `npm run lint` and fix the styles? Thanks!